release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -122,6 +122,51 @@ Previously, `-9223372036854775808` was allowed to be assigned to an integer in B
 int result = -9223372036854775808; // error: '9223372036854775808' is out of range for 'int'
 ```
 
+#### Restricted the typed binding of the query expression outer join to `var`
```

Comment:
Incomplete?

```suggestion
#### Restriction requiring the typed binding pattern of an outer join clause to use `var`
```

Reply:
Would the title length be fine @praneesha?

Reply:
@MaryamZi - Can we summarize this more as it is a title?

Reply:
Added some suggestions in https://github.com/ballerina-platform/ballerina-dev-website/pull/5812/files#r1023922325.